core.js

```diff
@@ -61,9 +61,11 @@ exports.init = function (dir, config) {
     }
 
     // delay startup a bit
-    setTimeout(() => {
-      SSB.net.conn.start()
-    }, 2000)
+    if (!config.core || !config.core.startOffline) {
```

Comment:
This is very hard to read. Maybe a variable: startOffline = config.core && config.core.startOffline. And then do the !startOffline in the if

Reply:
Makes sense.